content/bookings/intro.en.md

```diff
@@ -1,6 +1,5 @@
 # **Welcome!**
 
-For some of our activities, we kindly ask you to make a reservation.
-On this page, you can book a spot at one of our events.
+To join our Monday dinner, we kindly ask you to make a reservation.
```

Comment:
We may still use this form for other activities, so maybe keep it a bit more general